tests/utils/GraphicalUnitTester.py

```diff
@@ -135,7 +135,9 @@ def test(self, show_diff=False):
             if show_diff:
                 self._show_diff_helper(frame_data, expected_frame_data)
             assert test_result, (
-                f"The frames don't match. {str(self.scene).replace('Test', '')} has been modified."
+                f"The frames don't match. {str(self.scene).replace('Test', '')} has"
+                " been modified."
```

Comment:
This modification seems irrelevant to me. We have black here to do that work and we are comfortable with line length around 88. Please revert.

Reply:
This change was performed by black.

Reply:
You should run black from the root of the folder. Only then black will the configurations. Revert this, and all other things because of this.

Reply:
Your `pyproject.toml` doesn't contain any black configuration? Black will behave in the same way...

Reply:
[Black defaults to line length 88.](https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length) Then why does it break this? Are you sure you using black? Or does black uses any personal configuration?

Reply:
That line is originally 99c long. Black has wrapped it to fit within 88 characters

![Screenshot_20210123-144525_Chrome.jpg](https://user-images.githubusercontent.com/16487249/105589393-af6d2300-5d89-11eb-95e1-18fdaa9293f8.jpg)

Reply:
well now I think the test's folder isn't tested to be formatted I think...